modules/ROOT/pages/index.adoc

```diff
@@ -6,14 +6,15 @@ endif::[]
 
 APIkit is a toolkit that simplifies the development of API-led applications for Mule developers. It scaffolds applications from a given specification, validates inputs parameter validation, and creates xref:munit::munit-scaffold-test-task.adoc[MUnit test suites]. APIkit has a module for each supported web-service type:
 
+* xref:apikit-for-asyncapi.adoc[] supports AsyncAPI APIs.
 * xref:apikit-4-for-graphql.adoc[] supports GraphQL APIs. 
 * xref:apikit-4-for-odatav2.adoc[] and xref:apikit-4-for-odatav4.adoc[] support OData APIs. 
 * xref:apikit-4-for-rest.adoc[] supports RAML and OAS APIs.
 * xref:apikit-4-for-soap.adoc[] supports SOAP APIs. 
 
 With APIkit, incoming requests are intelligently routed to the appropriate endpoints based on the defined API contracts. This routing mechanism simplifies request handling, improves performance, and ensures that requests are processed by the correct API resources.
 
-Although each module is designed for a family of web services, all modules share common usage patterns. So if you are a Mule developer who works with SOAP APIs, you can easily work with GraphQL, RAML, OData, or OAS APIs.
+Although each module is designed for a family of web services, all modules share common usage patterns. So if you're a Mule developer who works with SOAP APIs, you can easily work with GraphQL, RAML, OData, or OAS APIs.
```

Comment:
Avoid "is designed". Maybe "works with"?

Suggest removing "easily" (sounds markety).

Reply:
OK! Thanks!